frontend/packages/pipelines-plugin/src/utils/hooks.ts

```diff
@@ -0,0 +1,10 @@
+import { gte } from 'semver';
+import { BadgeType, getBadgeFromType } from '@console/shared';
+import { usePipelineOperatorVersion } from '../components/pipelines/utils/pipeline-operator';
+
+export const usePipelineTechPreviewBadge = (namespace: string) => {
+  const operator = usePipelineOperatorVersion(namespace);
+  if (!operator) return null;
+  const installedGA = gte(operator.version, '1.4.0');
```

Comment:
I guess due to the lack of the other PRs merged, this is why we don't have constants/utilities.